run/multi-container/hello-php-nginx-sample/nginx/nginx.conf

```
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloudrun_mc_hello_php_nginx_serving_conf]
+server {
+    # Server at localhost
+    server_name localhost;
```

```suggestion
    server_name _;
```
Related to this post https://cloud.google.com/blog/products/serverless/cloud-run-now-supports-multi-container-deployments we don't name the server :)